metrics-report-generator/src/main/scala/it/pagopa/interop/metricsreportgenerator/util/Utils.scala

```diff
@@ -0,0 +1,42 @@
+package it.pagopa.interop.metricsreportgenerator.util
+import it.pagopa.interop.catalogmanagement.model.{CatalogDescriptor, CatalogItem, Draft}
+import it.pagopa.interop.commons.files.service.FileManager
+import it.pagopa.interop.commons.parser.{InterfaceParser, InterfaceParserUtils}
+import it.pagopa.interop.commons.utils.Digester
+import it.pagopa.interop.commons.utils.TypeConversions.EitherOps
+import it.pagopa.interop.metricsreportgenerator.models.{FileExtractedMetrics, Metric}
+
+import java.time.{OffsetDateTime, ZoneOffset}
+import scala.concurrent.{ExecutionContext, Future}
+object Utils {
+
+  final val defaultActivatedAt: OffsetDateTime = OffsetDateTime.of(2022, 12, 15, 12, 0, 0, 0, ZoneOffset.UTC)
```

Comment:
can we use the one defined in commons to avoid having different values?

Reply:
+1